### DISCUSSION TITLE
How do I password protect a page (using basic HTTP auth) in the app router?

### DISCUSSION THREAD

**Discussion by vedantroy** [vercel/next.js#50538] [Help] [answered]
### Summary

I want to password protect a random route in the app router. I don't want anything fancy, like Google login, etc.
I just want to use simple HTTP-based authentication.

What is the easiest way to do this?

**Comment by wwwdepot** [comment, marked as answer]
lazy internet, I googled it for you: `https://vancelucas.com/blog/how-to-add-http-basic-auth-to-next-js/`
